.env.example

```diff
@@ -2,6 +2,8 @@
 OPENAI_API_KEY=
 # RPC URL of the blockchain to fork from. Used for offline tx simulation.
 CHAIN_RPC_URL=
+# https://www.coingecko.com/ API Key
```

Comment:
For env variables related to specific agents,  I think they should (1) go below these 3 "main" env variables (2) be put in an agent-specific section.

We'll probably have more of these being added for the different agents, so setting up a way to organize env vars used by specific agents will pay off in the long run.

Reply:
Also (if we aren't already) we should probably check for the existence of these required env variables at the root of the agent's construction. For tests this is the classes constructor, and for the CLI it's the agent factory function.

Reply:
> Also (if we aren't already) we should probably check for the existence of these required env variables at the root of the agent's construction. For tests this is the classes constructor, and for the CLI it's the agent factory function.

yup agreed - i will add it in the agent constructor (since that cover both cases)